### PR TITLE
Create a function using a custom Node Docker image with Oracle DB Support

### PR DESCRIPTION
**Pull request by michael-w-williams**
This tutorial shows how to convert an existing Node image with Oracle DB support into an Fn function.

### CODE REVIEW

File: node/custom-db/README.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+(2) Change into the `hellodb` directory.
+![](images/userinput.png)
+>```
+> cd hellodb
+>```
+
+(3) Next, build your function.
+![](images/userinput.png)
+>```
+> fn -v build
```
Comment (shaunsmith) [inline node/custom-db/README.md]: You could skip the build since deploy will rerun it.

File: node/custom-db/hellodb/func.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+schema_version: 20180708
+name: hellodb
+version: 0.0.2
```
Comment (shaunsmith) [inline node/custom-db/hellodb/func.yaml]: set to 0.0.1

File: node/custom-db/hellodb/func.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+runtime: docker
+entrypoint: node func.js
+memory: 1024
```
Comment (shaunsmith) [inline node/custom-db/hellodb/func.yaml]: A GB seems like a lot.

File: node/custom-db/README.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+This tutorial requires the following:
+* Docker installed
+* Fn installed
+    * If you need help with Fn installation you can find instructions in the
+[Install and Start Fn Tutorial](../install/README.md).
+
+**Note:** Docker commands shown are for Linux. If you are using MacOS, the `sudo` command is not required.
+
+## Getting Started with Docker Image
+As a first step, let's create and run our starting Docker image.
```
[the remark this answers is not in the document]
Comment (michael-w-williams) [reply node/custom-db/README.md]: Fixed.

File: node/custom-db/README.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+```js
+schema_version: 20180708
+name: hellodb
+version: 0.0.1
+runtime: docker
+entrypoint: node func.js
+memory: 1024
+```
+
+The function name is `hellodb` so our code should be in a directory with that name. The runtime is set to `docker`. Again notice our node script is `func.js` in this case.
```
[the remark this answers is not in the document]
Comment (michael-w-williams) [reply node/custom-db/README.md]: Updated

File: node/custom-db/README.md (hunk 1 above)
Comment (michael-w-williams) [reply node/custom-db/README.md]: Removed.

File: node/custom-db/README.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+  "license": "MIT"
+}
+```
+
+Only one dependency here, the `oracledb` driver.
+
+Next the `index.js` is our actual Node program.
+```js
+const oracledb = require('oracledb')
+console.log(oracledb.versionString)
```
[the remark this answers is not in the document]
Comment (michael-w-williams) [reply node/custom-db/README.md]: Updated.

File: node/custom-db/hellodb/func.yaml (hunk 2 above)
Comment (michael-w-williams) [reply node/custom-db/hellodb/func.yaml]: Updated.

File: node/custom-db/hellodb/func.yaml (hunk 3 above)
Comment (michael-w-williams) [reply node/custom-db/hellodb/func.yaml]: Fixed.

**Comment by michael-w-williams** [comment]
Made all the requested fixes.

**Pull request merged**